source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -23,6 +23,18 @@ function isActionAllowed(action: Action, forbiddenActionTypes: Array<string>): b
 }
 
 export function createActionInstance(json: any, errors: Array<IValidationError>): Action {
+    if (!json["title"] && json["title"] !== "") {
+        raiseParseError(
+            {
+                error: Enums.ValidationError.ActionTitleCantBeNull,
```

Comment:
I think we can reuse PropertyCantBeNull here instead of defining a new enum?

Reply:
Thanks for pointing that out. I wasn't sure what exact error to raise